input_parser.star

```diff
@@ -663,10 +677,12 @@ def args_sanity_check(plan, deployment_stages, args, op_stack_args):
 
     # OP rollup deploy_optimistic_rollup and consensus_contract_type check
     if deployment_stages.get("deploy_optimism_rollup", False):
-        if args.get("consensus_contract_type", "cdk-validium") != "pessimistic":
+        user_consensus = user_args.get("args", {}).get("consensus_contract_type")
+        if user_consensus and user_consensus != "pessimitic":
```

Comment:
fyi `pessimitic` broke main for certain scenarios. @Freyskeyd  found and @jhkimqd  fixed (hopefully)